### ISSUE TITLE
Workflow documentation

### ISSUE THREAD

**Issue by jbarth-ubhd** [OCR-D/ocrd-website#321]
there is a `</code>` left in the workflow documentation:
![image](https://user-images.githubusercontent.com/30653779/183404334-416d7a8d-94c8-41e3-96b0-586e1b56eb85.png)

PS: are there any use cases for `pc-segmentation` ? As far as I remember this was the worst of all options a year ago during my tests.

**Comment by bertsky** [comment]
> are there any use cases for `pc-segmentation` ? As far as I remember this was the worst of all options a year ago during my tests.

Yes, and the dependencies were becoming too difficult to handle, that's why we disabled this by default in ocrd_all a while ago.

> there is a `</code>` left in the workflow documentation:

You can make changes to the WF guide here on the Wiki BTW – it gets synced back from time to time.
